forge/lib/tiler.py

```diff
@@ -226,59 +192,34 @@ def create(self):
         tilecount.value = 0
         skipcount.value = 0
 
-        jobs = Jobs(self.tmsConfig, self.t0)
-
-        if self.multiProcessing > 0:
-            pool = multiprocessing.Pool(NUMBER_POOL_PROCESSES, initWorker)
-
-            chunker = Chunks(jobs)
-
-            chunks = chunker.nextX(self.chunks)
-            bail = False
-            while len(chunks) > 0 and not bail:
-                logger.info('Processing %s jobs...' % str(len(chunks)))
-                async = pool.map_async(worker, chunks, 1)
-                try:
-                    while not async.ready():
-                        time.sleep(3)
-                except KeyboardInterrupt:
-                    logger.info('Keyboard interupt recieved, terminating workers...')
-                    pool.terminate()
-                    pool.join()
-                    bail = True
-                except Exception as e:
-                    logger.error('Error while generating the tiles: %s' % e)
-                    logger.error('Terminating workers...')
-                    pool.terminate()
-                    pool.join()
-                    raise Exception(e)
-                logger.info('Getting next jobs')
-                chunks = chunker.nextX(self.chunks)
-            if not bail:
-                logger.info('All jobs have been completed.')
-                logger.info('Closing processes...')
-                pool.close()
-                pool.join()
-        else:
-            for j in jobs:
-                try:
-                    worker(j)
-                except Exception as e:
-                    logger.error('An error occured while generating the tile: %s' % e)
-                    raise Exception(e)
+        tiles = Tiles(self.tmsConfig, self.t0)
+
+        pm = PoolManager()
+
+        useChunks = self.chunks
```

Comment:
I don't see why you need a new variable here, self.chunks should reflect the actual size of the chunks being used.